### PR TITLE
[Homework #4] - CSS by Antonina Vechorko

### PR DESCRIPTION
**Pull request by antoninavechorko**
@solvd-mmelnyk kindly check and share your feedback.

I tried to implement:
- semantically correct tags;
- good look with styles;
- BEM approach;
- dark mode and toggle switch using JS;

More functionality and responsive layout will be in new PR as part of Homework #5 that I'm planning to share soon.

Thanks you.

### CODE REVIEW

**Review by VitalikMelnyk — COMMENTED** [review]
Divide assets into some logical folders: images, icons, etc.

File: homeworks/antonina-vechorko/04-css/index.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css" integrity="sha512-oHDEc8Xed4hiW6CxD7qjbnI+B07vDdX7hEPTvn9pSZO1bcRqHp8mj9pyr+8RVC2GmtEfI2Bi9Ke9Ass0as+zpg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <header>
+        <div class="theme-switch">
+            <img src="assets/theme-switch.svg" class="theme-switch__icon" alt="Dark-theme">
+        </div>
+        <section class="person-container">
```
Comment (VitalikMelnyk) [inline homeworks/antonina-vechorko/04-css/index.html:17]: Fix the BEM approach.
See some helpful resources https://bem-cheat-sheet.9elements.com/

File: homeworks/antonina-vechorko/04-css/index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                            <img class="person__sm-icon icon3" src="assets/linkedin.svg" alt="LinkedIn">
+                        </a>
+                        <a href="https://www.youtube.com/" class="person__sm-link" target="blank">
+                            <img class="person__sm-icon icon4" src="assets/youtube.svg" alt="Youtube">
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <div class="achievements">
+                <div class="achievements-item">
```
Comment (VitalikMelnyk) [inline homeworks/antonina-vechorko/04-css/index.html:42]: Fix the BEM approach.
See some helpful resources https://bem-cheat-sheet.9elements.com/

File: homeworks/antonina-vechorko/04-css/index.html [hunk 3]
```diff
@@ -0,0 +1,197 @@
+<!doctype html>
```
Comment (VitalikMelnyk) [inline homeworks/antonina-vechorko/04-css/index.html:1]: Please, review your BEM approach.

**Comment by antoninavechorko** [comment]
@VitalikMelnyk thank you for feedback.
In the last commit the following changes were made:
- assets folder was divided by icons and images folders
- BEM naming was fixed.

Kindly recheck if all is ok now.

**Review by VitalikMelnyk — APPROVED** [review]
Well Done @antoninavechorko!
Approved.

**Pull request merged**